<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unitssettings</class>
 <widget class="QMainWindow" name="unitssettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Единицы измерений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="dimsLabel">
        <property name="text">
         <string>Размеры:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="dimsCombo">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>мкм</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>мм</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>см</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>м</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="freqsLabel">
        <property name="text">
         <string>Частота:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="freqsCombo">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Гц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>КГц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>МГц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГГц</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Время:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="timeCombo">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>пс</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>нс</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>мкс</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>мс</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>с</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="okButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ок</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>186</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
